docs/content.md

```diff
@@ -79,6 +78,20 @@ If you can't find a corresponding file in the [/app/views/content](https://githu
 
 If you are looking to edit content associated with a form element in particular (for example, a label for a text input) then  you should look in the [translations file](https://github.com/DFE-Digital/get-into-teaching-app/blob/master/config/locales/en.yml). It's important to only change the text content in this file and not the Yaml keys that identify the content.
 
+## Tone of voice
+
+It’s important that the content on Get Into Teaching uses a consistent tone of voice.
+
+When writing new content, you can look at the language we use on other pages to help keep it consistent. For example, we always refer to children in schools as pupils, rather than students.
+
+Some useful resources include:
+
+* GIT brand book and messaging matrix (available on Sharepoint)
+* [accessibility content guidance](#accessibility)
+* [GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z)
+* [Department for Education style guide](https://design.education.gov.uk/design-system/style-guide)
+
+In some cases, policy teams may also ask us to word things in a certain way or include specific information. 
```

Comment:
Should we say something like 'Unless there are commercial reasons why we need to use certain language, we can amend requested wording as needed to be consistent with the Get Into Teaching tone of voice,"

cc @Sarah-DfE